config/core/configmaps/leader-election.yaml

```diff
@@ -57,10 +57,4 @@ data:
     retryPeriod: "2s"
     # enabledComponents is a comma-delimited list of component names for which
     # leader election is enabled. Valid values are:
-    #
-    # - controller
-    # - hpaautoscaler
-    # - certcontroller
-    # - istiocontroller
-    # - nscontroller
-    enabledComponents: "controller,hpaautoscaler,certcontroller,istiocontroller,nscontroller,webhook"
+    enabledComponents: "controller,contour-ingress-controller,hpaautoscaler,certcontroller,istiocontroller,net-http01,nscontroller,webhook"
```

Comment:
(lacking context) why is this not an array?

Reply:
configmaps are `map[string]string`